src/index.ts

```diff
@@ -66,6 +66,19 @@ export class Collection<K, V> extends Map<K, V> {
 		return super.clear();
 	}
 
+	/**
+	 * Gets an element if the key exists, otherwise sets it to {@param defaultValue} and returns the {@param defaultValue}.
+	 * @param {*} key - Key to get from/set to the collection.
```

Comment:
```suggestion
	 * @param {*} key - Key to ensure from the collection.
```

Although I'm not 100% about this wording, cc: @vladfrangu @iCrawl

Reply:
I was thinking something closer to "Key to ensure *in* the collection", because this method ensures that the key exists in the collection